query_graphql.py

```diff
@@ -8,7 +9,7 @@
 async def query_image():
     # Select your transport with a defined url endpoint
     transport = AIOHTTPTransport(
-        url="https://painmon-api.herokuapp.com/graphql", headers={"content-type": "application/json"})
+        url="https://f9fc-118-136-96-42.ap.ngrok.io/graphql", headers={"content-type": "application/json"})
```

Comment:
Can you put this into a constant, so its easier to change it later? Something like
```suggestion
        url=BASE_URL, headers={"content-type": "application/json"})
```
```